src/components/AppStepper/FeaturesStep/FeaturesStep.js

```diff
@@ -252,23 +253,34 @@ class FeaturesStep extends Component {
           <Typography>
             <FormattedMessage id="stepFeaturesDesc" />
           </Typography>
-          <div className={classes.actionsContainer}>
-            {availableFeatures.map(
-              (item) =>
-                item.show &&
-                (item.boards.includes(board.name) ||
-                  item.boards.includes('all')) && (
-                  <FeaturesSelector
-                    classes={classes}
-                    // value={this.state[item.name]}
-                    value={tempState[item.name]}
-                    item={item}
-                    onChange={this.handleChangeCheckBox}
-                    key={item.name}
-                  />
-                )
-            )}
-          </div>
+          {featureTypes.map(
+            (type) => (
+              <>
```

Comment:
Use div with `className` to add additional spacing. If you are using map on arry then you have to use key on each element in JSX:
```
          <Typography>
            <FormattedMessage id="stepFeaturesDesc" />
          </Typography>
          {featureTypes.map((type) => (
            <div className={classes.featuresGroup} key={type}>
              <Typography>
                <FormattedMessage
                  id={
                    'stepFeatures' +
                    type.charAt(0).toUpperCase() +
                    type.slice(1) +
                    'TypeDesc'
                  }
                />
              </Typography>
              <Divider />
              <div className={classes.actionsContainer}>
                {availableFeatures.map(
                  (item) =>
                    item.show &&
                    item.type === type &&
                    (item.boards.includes(board.name) ||
                      item.boards.includes('all')) && (
                      <FeaturesSelector
                        classes={classes}
                        // value={this.state[item.name]}
                        value={tempState[item.name]}
                        item={item}
                        onChange={this.handleChangeCheckBox}
                        key={item.name}
                      />
                    )
                )}
              </div>
            </div>
```

Reply:
you need also this in styles.js:
```
featuresGroup: {
    marginTop: theme.spacing.unit * 2,
  },
```

Reply:
Thanx for teaching me. Thats why I need a workaround with fragments.